loaf-explainer.md

```diff
@@ -6,21 +6,26 @@ This is work in progress. Feedback welcome, lots of things might change etc.
 
 ## Overview
 
-"Jankiness" - the feeling that "this page is not responsive to interactions", is a common problem for users on the web today.
-By introducing [INP](https://web.dev/inp/) into Core Web Vitals, we hope that authors can have a better indication as to
-how their pages are doing in that regard. But INP shows you the effect, but not the cause of the jankiness.
+"Sluggishness" - the feeling that "this page is not responsive to interactions", is a common problem for users on the web today.
+By introducing [INP](https://web.dev/inp/) into Core Web Vitals, we hope that authors can have a better indication as to how their pages are doing in that regard. But INP shows you the effect, but not the cause of the sluggishness.
 
-Long animation frames (LoAF), as a revamp of `longtasks`, aim to assist with that: a LoAF is an indication that in a particular
-point in time, the browser was busy, in such a way that it took a long time from the beginning of a task until updating the rendering
-(or until the point where it was clear that no render was necessary).
+Long animation frames (LoAF), as a revamp of `longtasks`, aim to assist with that: a LoAF is an indication that in a particular point in time, the browser was congested, in such a way that it took a long time from the beginning of a task until updating the rendering (or until the point where it was clear that no render was necessary).
 
 Since having busy ("LoAF-heavy") sequences can potentially cause delayed response to interactions, and the LoAF entries themselves contain information about
-what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose
-for this type of performance issue.
+what was blocking, e.g. long scripts or layout, LoAF can potentially become a powerful tool, enabling the use of real-user monitoring (RUM) to diagnose for this type of performance issue.
+
+## Value to End Users & Publishers
+
+Several case studies like [The Economic times](https://web.dev/economic-times-inp/#how-has-improving-inp-helped-the-economic-times) and [RedBus](https://web.dev/redbus-inp/#business-impact), that improving page responsiveness via the INP metric directly helps with user delight and has a business impact.
```

Comment:
I think this sentence is missing a verb -- "have shown that improving page responsiveness", or "suggest that improving page responsiveness", maybe?

Reply:
Fixed